tests/e2e/support/commands.js

```diff
@@ -412,6 +424,9 @@ Cypress.Commands.add('playPointsSpectator', (card, pNum) => {
 });
 
 Cypress.Commands.add('playPointsById', (cardId) => {
+  if (Cypress.env('CYPRESS_USE_GAMESTATE_API')) {
+    throw new Error(gameStateApiError);
+  }
```

Comment:
I think it may make sense to make a function similar to `makeSocketRequest` in the gamestore for these API calls instead of checking for an error like this.

It may be out of scope on this specific PR though

Curious on your thoughts @itsalaidbacklife , @seriouslysean

Reply:
If I’d just for testing a specific action wit would be kinda weird. You could have a conditional action that’s only there for testing maybe?

Either way I agree with you though, to keep things clean. Definitely out of scope :) — separate your concerns.

Reply:
#974